source/features/bypass-checks.tsx

```diff
@@ -46,10 +47,6 @@ void features.add(import.meta.url, {
 	include: [
 		pageDetect.isRepo,
 	],
-	exclude: [
-		pageDetect.isEmptyRepo,
```

Comment:
It doesn't work with `awaitDomReady: false` and it's not needed with the observer